code/datums/status_effects/debuffs.dm

```diff
@@ -289,6 +294,10 @@
 /datum/status_effect/cling_tentacle/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_IMMOBILIZED, "[id]")
 
+/datum/status_effect/cling_tentacle/batterer
+	id = "cling_tentacle"
```

Comment:
You might want to make this a different ID?

Reply:
ye